proto/v1beta1/swagger/merged/grafeas.swagger.json

```diff
@@ -1450,6 +1556,29 @@
       },
       "description": "A note that indicates a type of analysis a provider would perform. This note\nexists in a provider's project. A `Discovery` occurrence is created in a\nconsumer's project at the start of analysis."
     },
+    "googlerpcStatus": {
```

Comment:
I am curious what caused the change of the rpcStatus to googlerpcStatus.

Reply:
I don't know how this happened tbh. I regenerated everything and it looks good now.